salt/proxy/nxos.py

```diff
@@ -317,7 +316,7 @@ def sendline(command, method='cli_show_ascii', **kwargs):
     return result
 
 
-def proxy_config(commands, **kwargs):
+def proxy_config(commands, no_save_config=None):
```

Comment:
`no_save_config=None`, then we negate it everywhere.

`save_config` defaulting to True seems to make more sense logically. 

Do you see any issues with us flipping the logic?

Reply:
@mikewiebe, do you see any issues with flipping the logic?

Reply:
To be clear, I'm happy to do the flipping, and am just asking if you see problems with the flip.

Reply:
I am fine with the flip.

Reply:
done